.gitlab-ci.yml

```diff
@@ -23,9 +23,8 @@ include:
 variables:
   ASAP_HOSTNAME: hub.asap.science
   PLAYWRIGHT_BROWSERS_PATH: 0
-  BASE_NODE_IMAGE: $CI_REGISTRY/$CI_PROJECT_PATH/node:14.17-alpine
 
-image: $BASE_NODE_IMAGE
+image: node:14.17-alpine
```

Comment:
we might run into docker limits again with this, no?

Reply:
Now reading this from ECR.